src/app/pages/datasets/components/data-protection-card/data-protection-card.elements.ts

```diff
@@ -0,0 +1,33 @@
+import { marker as T } from '@biesbjerg/ngx-translate-extract-marker';
+import { UiSearchableElement } from 'app/modules/global-search/interfaces/ui-searchable-element.interface';
+
+export const dataProtectionCardElements = {
+  hierarchy: [T('Datasets')],
+  anchorRouterLink: ['/datasets'],
+  elements: {
+    createSnapshot: {
+      hierarchy: [T('Create Snapshot')],
+      synonyms: [T('Take Snapshot')],
+    },
+    manageSnapshots: {
+      hierarchy: [T('Manage Snapshots')],
+      synonyms: [T('Snapshot Manager')],
+    },
+    manageSnapshotTasks: {
+      hierarchy: [T('Manage Snapshots Tasks')],
+      synonyms: [T('Snapshot Task Manager')],
+    },
+    manageReplicationTasks: {
+      hierarchy: [T('Manage Replication Tasks')],
+      synonyms: [T('Replication Task Manager')],
+    },
+    manageCloudSyncTasks: {
+      hierarchy: [T('Manage Cloud Sync Tasks')],
```

Comment:
I don't think we need to add those because it's the same as Data Protection >  Cloud Sync Tasks.